app/views/track/_tracking_links_simple.html.erb

```diff
@@ -0,0 +1,15 @@
+<%
+  existing_track = local_assigns.fetch(:existing_track) do
+    TrackThing.find_existing(@user, track_thing) if @user
+  end
+%>
+
+<% if existing_track %>
```

Comment:
The [current version](https://github.com/mysociety/alaveteli/blob/develop/app/views/track/_tracking_links.html.erb#L7-L10) accounts for the request being your own. I guess we'd need to do something similar.